src/Panel/Panel.php

```diff
@@ -109,32 +602,260 @@ public static function render(App $kirby)
         // get the uri object for the panel url
         $uri = new Uri($url = $kirby->url('panel'));
 
+        // inject globals
+        $globals = static::globals($kirby);
+        $fiber   = array_merge_recursive(A::apply($globals), $fiber);
+
         // fetch all plugins
         $plugins = new Plugins();
 
-        $view = new View($kirby->root('kirby') . '/views/panel.php', [
-            'kirby'     => $kirby,
-            'config'    => $kirby->option('panel'),
-            'assetUrl'  => $kirby->url('media') . '/panel/' . $kirby->versionHash(),
-            'customCss' => static::customCss($kirby),
-            'icons'     => static::icons($kirby),
-            'pluginCss' => $plugins->url('css'),
-            'pluginJs'  => $plugins->url('js'),
-            'panelUrl'  => $uri->path()->toString(true) . '/',
-            'nonce'     => $kirby->nonce(),
-            'options'   => [
-                'url'         => $url,
-                'site'        => $kirby->url('index'),
-                'api'         => $kirby->url('api'),
-                'csrf'        => $kirby->option('api.csrf') ?? csrf(),
-                'translation' => 'en',
-                'debug'       => $kirby->option('debug', false),
-                'search'      => [
-                    'limit' => $kirby->option('panel.search.limit') ?? 10
-                ]
+        return new Response(
+            Tpl::load($kirby->root('kirby') . '/views/panel.php', [
+                'assets'   => static::assets($kirby),
+                'icons'    => static::icons($kirby),
+                'nonce'    => $kirby->nonce(),
+                'fiber'    => $fiber,
+                'panelUrl' => $uri->path()->toString(true) . '/',
+            ])
+        );
+    }
+
+    /**
+     * Router for the Panel views
+     *
+     * @param \Kirby\Cms\App $kirby
+     * @param string $path
+     * @return \Kirby\Http\Response|false
+     */
+    public static function router(App $kirby, string $path = null)
+    {
+        if ($kirby->option('panel') === false) {
+            return false;
+        }
+
+        // switch off pagination exceptions
+        Pagination::$validate = false;
```

Comment:
Why do we need this here? I'm sure there is a reason, a comment that explains it would be great (the current one is stating the obvious).

Reply:
Tbh, this originated even from the 2020 Inertia trial. I just tried without it and all pagination seems still to work. Let's remove it and if we actually encounter a reason bring it back with good documentation. Or if @bastianallgeier remembers what it was for, please yell :D